contracts/AnchoredView/AnchoredView.sol

```diff
@@ -0,0 +1,254 @@
+    uint constant blocksInADay = 5760;
+
+    /// @notice The event emitted when the median price is updated
+    event PriceUpdated(string symbol, uint price);
+
+    /// @notice The event emitted when new prices are posted but the stored price is not updated due to the anchor
+    event PriceGuarded(string symbol, uint reporter, uint anchor);
+
+    /// @notice The event emitted when reporter invalidates itself
+    event ReporterInvalidated(address reporter);
+
+    /// @notice The event emitted when the anchor is cut for staleness
+    event AnchorCut(address anchor);
+
+    /**
+     * @param data_ Address of the Oracle Data contract
+     * @param reporter_ The reporter address whose price will be used if it matches the anchor
+     * @param anchor_ The PriceOracleProxy that will be used to verify reporter price, or serve prices not given by the reporter
+     * @param anchorToleranceMantissa_ The tolerance allowed between the anchor and median. A tolerance of 10e16 means a new median that is 10% off from the anchor will still be saved
+     * @param tokens_ The CTokens struct that contains addresses for CToken contracts
+     */
+    constructor(OpenOraclePriceData data_,
+                address reporter_,
+                AnchorOracle anchor_,
+                uint anchorToleranceMantissa_,
+                CTokens memory tokens_) SymbolConfiguration(tokens_) public {
+        reporter = reporter_;
+        anchor = anchor_;
+        priceData = data_;
+
+        require(anchorToleranceMantissa_ < 100e16, "Anchor Tolerance is too high");
+        upperBoundAnchorRatio = 100e16 + anchorToleranceMantissa_;
+        lowerBoundAnchorRatio = 100e16 - anchorToleranceMantissa_;
+    }
+
+    /**
+     * @notice Post open oracle reporter prices, and recalculate stored price by comparing to anchor
+     * @dev We let anyone pay to post anything, but only prices from configured reporter will be stored in the view
+     * @param messages The messages to post to the oracle
+     * @param signatures The signatures for the corresponding messages
+     * @param symbols The symbols to compare to anchor for authoritative reading
+     */
+    function postPrices(bytes[] calldata messages, bytes[] calldata signatures, string[] calldata symbols) external {
+        require(messages.length == signatures.length, "messages and signatures must be 1:1");
+
+        // Save the prices
+        for (uint i = 0; i < messages.length; i++) {
+            priceData.put(messages[i], signatures[i]);
+        }
+
+        // load usdc for using in loop to convert anchor prices to dollars
+        uint usdcPrice = readAnchor(cUsdcAddress);
+
+        // Try to update the view storage
+        for (uint i = 0; i < symbols.length; i++) {
+            CTokenMetadata memory tokenConfig = getCTokenConfig(symbols[i]);
+            // symbol is not supported in the view, but allow writing to data
+            if (tokenConfig.cTokenAddress == address(0)) continue;
```

Comment:
is this just to explicity call out this case? doesnt impact control flow at all, right?

Reply:
the call will eventually revert if the token is not configured, so this is to short circuit before that happens

Reply:
so, yes, it does influence control flow by skipping the innards of this loop for this tokenConfig